src/macosx/hidjoy.m

```diff
@@ -765,17 +765,18 @@ static bool get_joystick_active(ALLEGRO_JOYSTICK *joy_)
 
 #endif // Leopard+
 
+#ifndef NSAppKitVersionNumber10_5
+#define NSAppKitVersionNumber10_5 949
```

Comment:
The constants are not defined when compiled using an earlier or equal version of the NSAppKit. For example, on NSAppKit 10.5, the constant above will not be defined. On 10.10  NSAppKitVersionNumber10_10 is not  defined. At least  that is what I understood from the dev docs and forum threads I read.